build.sbt

```diff
@@ -4268,6 +4274,8 @@ lazy val `engine-runner` = project
   .dependsOn(`runtime-version-manager`)
   .dependsOn(`logging-service`)
   .dependsOn(`logging-service-logback` % Runtime)
+  .dependsOn(`logging-service-telemetry` % Runtime)
+  .dependsOn(`logging-service-opensearch` % Runtime)
```

Comment:
- @Akirathan, the goal of #13720 was to unify building of JVM & NI
- true, now we are using module path in both cases, but:
- the module path configuration is still duplicated, right?
- JVM needs this [componentModulePath](https://github.com/enso-org/enso/pull/14272/files#r2510916272)
- while NI is using the _path listed here_
- shouldn't the `enso` NI also set `--module-path component` directory?

Reply:
Module path for NI is set from `engine-runner/Runtime/modulePath` https://github.com/enso-org/enso/blob/3f00d276c0d6d453ad421348fc147958adc038ae/build.sbt#L4164, which is probably different from `componentModulePath`. Could we unify it? Probably. I have not tried it before.